regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -46,6 +47,16 @@ var PreambleView = ChildView.extend({
     );
   },
 
+  openCitation: function(e) {
+    var $target = $(e.currentTarget);
+    var hash = $target.attr('href');
```

Comment:
Say you're in section I and you see a link to section III.B. Right now, the markup generates a link like:
```
href="#0000_0000-III-B"
```

That's not accurate as that anchor doesn't exist on this page. In particular it _won't_ work for non-JS users. The markup should be closer to:

```
href="/preamble/0000_0000/III##0000_0000-III-B"
```

This JS relies on the incorrect behavior, if that makes sense?